app/src/components/HomePage/HomePage.tsx

```diff
@@ -204,7 +204,7 @@ export default function HomePage({
                       {[
                         "tab-emission-inventory-calculation-title",
                         "tab-emission-inventory-results-title",
-                        "tab-cap-title",
+                        ...(inventory?.city?.country === "Brazil" ? ["tab-cap-title"] : []),
```

Comment:
Ideally we have a list of countries that the CAP feature is enabled for (or specific projects/ organizations in our database that have a boolean flag set to enable it). Works for now, but let's try to remove it in a future refactoring step.

Reply:
absolutely, when we migrate this to a DB we can use the locodes to show the tab only for those cities that have actions